doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -180,6 +180,10 @@ Requirements to software and settings:
 
         sudo locale-gen en_US.UTF-8
 
+-   Ensure that the packages `locales` and `software-properties-common` are present
+
+        sudo apt-get install locales software-properties-common
+
 
```

Comment:
This bullet point could be before the previous one, since the command `locale-gen` will fail if `locales` is not installed.
Also, version history and the document version should be updated.